bin/compile

```diff
@@ -47,8 +47,6 @@ handleDefaultPkgSpec() {
                 warn "To install a different package spec add a comment in the following form to your \`go.mod\` file:"
                 warn "// +heroku install ./cmd/..."
                 warn ""
-                warn "For more details see: https://devcenter.heroku.com/articles/go-apps-with-modules#build-configuration"
```

Comment:
I was going to say that this seemed useful, because there are lots of build details to customize to get things really working well for Go.. but the decventer article is empty. So 👍 at least until we've got better docs in place.